db/migrate/20160915085131_create_users_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsersTable < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.string :password_digest
+      t.integer :tweet_ids
```

Comment:
If you look in your database, what is stored in `tweet_ids` column? What is it used for? 
In a has_many, belongs_to relationship, which model has the foreign key?